src/library_browser.js

```diff
@@ -225,6 +225,32 @@ var LibraryBrowser = {
       };
       Module['preloadPlugins'].push(audioPlugin);
 
+#if WASM
+      var wasmPlugin = {};
```

Comment:
please also put this behind the `MAIN_MODULE` flag, so it's not present in code that doesn't support shared modules.